runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesOverlay.java

```diff
@@ -47,30 +56,37 @@
 		setPriority(OverlayPriority.LOW);
 		this.plugin = plugin;
 		this.config = config;
+		panelComponent.setOrientation(PanelComponent.Orientation.HORIZONTAL);
+		panelComponent.setWrapping(4);
 	}
 
+	private final PanelComponent panelComponent = new PanelComponent();
```

Comment:
For moving stuff below injected fields i meant below fields not also below constructor......